apps/ledger-live-mobile/src/components/useExportLogs.tsx

```diff
@@ -8,7 +8,9 @@ export default function useExportLogs() {
   return useCallback(() => {
     const exportLogs = async () => {
       const logs = logReport.getLogs();
-      const base64 = Buffer.from(JSON.stringify(logs)).toString("base64");
+      const base64 = Buffer.from(JSON.stringify(logs, null, 4)).toString(
```

Comment:
Just here if you like my preference so that you can commit from UI
```suggestion
      const base64 = Buffer.from(JSON.stringify(logs, null, 2)).toString(
```